### DISCUSSION TITLE
Add PowerOnState Blink

### DISCUSSION THREAD

**Discussion by Garywoo** [arendst/Tasmota#11306] [Q&A] [answered]
I would find it very useful if there was an option to trigger a blink upon power up.

My use case is using a Sonoff RE5V1C to turn on a Divoom Pixoo display, when power is applied, by way of my PC turning on, and powering up it's USB ports. By default the display does not turn on when power is applied, so I want to use the Sonoff to 'press' the power button for a couple of seconds to turn it on, when power is applied.

**Comment by ascillato** [comment]
Hi, that is already supported. Just use rules to make the behavior you want. Please, check the docs at rules. There are plenty of examples there.

**Comment by barbudor** [comment, marked as answer]
Blink is not the best solution as it is intended to perform multiple on/off cycle (although you can set the blickcount to 1).
You should use `PulseTime` with a short value so when performing `Power ON`, Tasmota will pulse ON the output for the configured time and automatically turn to off.
Once Pulsetime is configured, you can use a rule such as `on system#boot do power on endon` to automatically trigger the pulse.

As pointed by Adrian, read the [Rules](https://tasmota.github.io/docs/Rules/) section of the documentation, multiple time, in both directions :)

**Reply by Garywoo** [reply]
`PulseTime` was indeed a better solution. I used this in conjunction with `PowerOnState 1` to toggle the output when the device is powered up. 

No requirement for rules here, although they do look pretty powerful and good to keep in mind.
